### DISCUSSION TITLE
ISR seems to break with fast refreshing with SvelteKit

### DISCUSSION THREAD

**Discussion by consultingninja** [vercel/community#3310] [Bug Report]
### Page to Investigate

https://vercel.com/docs/concepts/incremental-static-regeneration

### Steps to Reproduce

ISR documentation says the data will only be re-validated only as often as the ISR property says in the config object.  

With ISR enabled and set to 10 seconds and my client page set to re-navigate to itself every 10ms the ISR doesn't appear to work.  You can see a video of the issue on my YouTube channel here: https://youtu.be/NolSFhEDsI0 the problem is shown at 6:33 timestamp.  I believe what is happening is that the platform releases the lock and all the client requests flood through and allow many requests before being locked in again.  I can see some potential issues with this, and it probably deserves a look.

To replicate simply deploy a SvelteKit app with one route that has ISR property set in the config object, this route should have a load function that pings the random user api for a random user and displays that users information.  In your page.svelte markup add (I know this is probably not a widely used use case) a timeout that runs every 10 ms that re-navigates to the same page.

Notice ISR seems to break intermittently.  Locking then unlocking etc. You can use my repository if you would like.

https://github.com/consultingninja/IsrExample

Hope this is helpful!
Mitch Migala aka Consulting Ninja

**Comment by amyegan** [comment]
Thanks for letting us know! This post has been shared with the appropriate team for investigation 🙇
